<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BloodRequestClass</class>
 <widget class="QMainWindow" name="BloodRequestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blood Bank Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RecipientReg.qrc">
    <normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="RecipientReg.qrc">:/RecipientReg/profile.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="accountF">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>35</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/RecipientReg/—Pngtree—includes set of avatars icons_8537422 (1).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
border-style:solid;
border-width:0px;
border-color: rgb(220, 3, 6);
border-radius:9px;
color: white;
font: 700 8pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="accountM">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/RecipientReg/—Pngtree—includes set of avatars icons_8537422 (2).png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="request">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(240, 240, 240);
border-style:solid;
border-width:2px;
border-color: rgb(220, 3, 6);
border-radius:25px;
color: rgb(220, 3, 6);
font: 700 8pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="pop_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/RecipientReg/request.png)
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ok">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/RecipientReg/Rectanglered.png);
border-style:solid;
border-width:2px;
border-color: rgb(220, 3, 6);
border-radius:20px;
color: white;
font: 700 8pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="quant">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-style:solid;
border-width:1px;
border-color: rgb(106, 106, 106);
border-radius:2px;
color:rgb(0, 0, 0);
font: 700 8pt &quot;Arial&quot;;</string>
    </property>
    <property name="minimum">
     <double>0.250000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="hos">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-style:solid;
border-width:1px;
border-color: rgb(106, 106, 106);
border-radius:8px;
color: black;
font: 700 8pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Here Some Tips Before Recieving Blood :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ask questions: Before accepting a blood transfusion, ask your healthcare provider
about the risks and benefits of the procedure. Make sure you fully understand why 
a transfusion is necessary and what to expect.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Consider alternatives: In some cases, there may be alternatives to a blood 
transfusion, such as medications or other treatments. Ask your healthcare 
provider about these options and if they are appropriate for your situation.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Prepare for the procedure: If you decide to go ahead with a blood transfusion, 
make sure you are well-rested and hydrated before the procedure. Follow any 
instructions provided by your healthcare provider, such as fasting or stopping 
certain medications.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hospital</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>How much blood you will recieve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:transparent</string>
    </property>
    <property name="text">
     <string>Blood request completed,Thank you.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/RecipientReg/pngwing.com (57).png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(104, 104, 104)</string>
    </property>
    <property name="text">
     <string>Previous Page</string>
    </property>
   </widget>
   <zorder>name</zorder>
   <zorder>request</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>hos</zorder>
   <zorder>quant</zorder>
   <zorder>pop_2</zorder>
   <zorder>label_7</zorder>
   <zorder>accountM</zorder>
   <zorder>accountF</zorder>
   <zorder>logout</zorder>
   <zorder>user</zorder>
   <zorder>back</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RecipientReg.qrc"/>
 </resources>
 <connections/>
</ui>
